src/Life/Github.hs

```diff
@@ -74,8 +87,11 @@ pushRepo :: Text -> IO ()
 pushRepo = insideRepo . askToPushka
 
 -- | Clones @dotfiles@ repository assuming it doesn't exist.
-cloneRepo :: Owner -> IO ()
-cloneRepo (Owner owner) = do
+cloneRepo :: Maybe Owner -> IO ()
+cloneRepo mo = do
+    owner <- case mo of
+        Nothing -> getUsername
+        Just (Owner o) -> pure o
```

Comment:
Let's move it in a separate function called `getOwnerLogin :: Maybe Owner -> IO Text`. It seems be used in several places so we can remove a bit of code duplication. Also, I believe it can be written a bit simpler like `maybe getUsername (pure . unOwner)`